yacc.go

```diff
@@ -3470,22 +3469,20 @@ func $$Parse($$lex $$Lexer) int {
 
 func ($$rcvr *$$ParserImpl) Parse($$lex $$Lexer) int {
 	var $$n int
-	var $$lval $$SymType
```

Comment:
FYI, these changes were driven by the memory profiler. It will show you the line allocations are occurring on so you don't have to guess about whether escape analysis is placing something on the stack or on the heap.